src/common/i18n/translations/en.json

```diff
@@ -0,0 +1,362 @@
+    },
+    "wrongFormatQRScanAgain": {
+      "title": "Wrong format",
+      "body": "Scan QR code again or get a new QR code from your in-charge.",
+      "primaryActionText": "OK"
+    },
+    "wrongFormatValidCode": {
+      "title": "Wrong format",
+      "body": "Enter valid code details.",
+      "primaryActionText": "OK"
+    },
+    "wrongFormatDeviceCode": {
+      "title": "Wrong format",
+      "body": "Scan device code again or get a new device from your in-charge.",
+      "primaryActionText": "OK"
+    },
+    "wrongFormatID": {
+      "title": "Wrong format",
+      "body": "Enter or scan a valid ID number.",
+      "primaryActionText": "OK"
+    },
+    "wrongFormatQRGetNew": {
+      "title": "Wrong format",
+      "body": "Get a new QR code from your in-charge.",
+      "primaryActionText": "OK"
+    },
+    "alreadyUsedCode": {
+      "title": "Already Used",
+      "body": "Enter unique code details",
+      "primaryActionText": "OK"
+    },
+    "alreadyUsedItem": {
+      "title": "Already Used",
+      "body": "Scan another item that is not tagged to any ID number",
+      "primaryActionText": "OK"
+    },
+    "wrongFormatCode": {
+      "title": "Wrong format",
+      "body": "Enter or scan valid code details",
+      "primaryActionText": "OK"
+    },
+    "incompleteEntryCode": {
+      "title": "Incomplete entry",
+      "body": "Enter or scan code details",
+      "primaryActionText": "OK"
+    },
+    "wrongFormatNotValidDeviceCode": {
+      "title": "Wrong format",
+      "body": "Scan valid device code",
+      "primaryActionText": "OK"
+    },
+    "incompleteEntryScanDeviceCode": {
+      "title": "Incomplete entry",
+      "body": "Scan device code",
+      "primaryActionText": "OK"
+    },
+    "wrongFormatContactNumber": {
+      "title": "Wrong format",
+      "body": "Enter valid contact number",
+      "primaryActionText": "OK"
+    },
+    "insufficientQuota": {
+      "body": "Insufficient quota"
+    },
+    "invalidQuantity": {
+      "body": "Invalid quantity"
+    },
+    "categoryDoesNotExist": {
+      "body": "Category does not exist"
+    },
+    "systemErrorServerIssues": {
+      "title": "System error",
+      "body": "We are currently facing server issues. Try again later or contact your in-charge if the problem persists",
+      "primaryActionText": "OK"
+    },
+    "discardTransaction": {
+      "title": "Discard transaction?",
+      "body": "This will clear all scanned items",
+      "primaryActionText": "Cancel",
+      "secondaryActionText": "Discard"
+    },
+    "errorScanning": {
+      "title": "Error scanning",
+      "primaryActionText": "Continue Scanning"
+    },
+    "scanLimitReached": {
+      "title": "Scan limit reached",
+      "primaryActionText": "OK"
+    },
+    "removeItem": {
+      "title": "Remove item?",
+      "body": "Do you want to remove this item: %{voucherSerial}?",
+      "primaryActionText": "Cancel",
+      "secondaryActionText": "Remove"
+    }
+  }
```

Comment:
Shall we leave this out of this PR, and put them in the other MR for error messages?